renku/core/commands/format/datasets.py

```diff
@@ -27,9 +28,17 @@ def tabular(client, datasets, *, columns=None):
     if not columns:
         columns = "id,date_created,name,creators,tags,version"
 
+    _create_dataset_short_description(datasets)
+
     return tabulate(collection=datasets, columns=columns, columns_mapping=DATASETS_COLUMNS)
 
 
+def _create_dataset_short_description(datasets):
+    for dataset in datasets:
+        lines = textwrap.wrap(dataset.description, width=64, max_lines=5) if dataset.description else []
```

Comment:
Could we calculate the width by using the `COLUMNS` env variable (and falling back to 64 if it's not set)? like `width = $COLUMNS - <width of other columns>`?

Reply:
The problem here is that we don't have the width of other columns and `tabulate` cannot set column width as well.